src/main/java/ledger/controller/DbController.java

```diff
@@ -446,6 +450,29 @@ public File getDbFile() {
         return new TaskWithArgs<>(db::addBalanceForAccount, balance);
     }
 
+    /**
+     * Creates a Task that can be used to make an asynchronous call to the database to check transactions as a batch with Ignore checks
+     *
+     * @param transactions List of transactions to add to the database
+     * @return A task for the asynchronous call
+     */
+    public TaskWithArgsReturn<List<Transaction>, List<Transaction>> batchTransactionIgnoreCheck(List<Transaction> transactions) {
```

Comment:
Ill get rid of the unused code. It was used in an earlier commit and I didnt remove it.